ci/Dockerfile

```
@@ -11,20 +11,20 @@ RUN yum install -y \
       glibc-langpack-en \
       git \
       make \
-      python39 \
-      python39-devel \
-      python39-pip \
-      python39-setuptools \
+      python3 \
+      python3-devel \
+      python3-pip \
+      python3-setuptools \
       gcc \
       openldap-devel \
- && pip3 install --no-cache-dir --upgrade setuptools pip \
- && pip3 install --no-cache-dir \
+ && python3.9 -m pip install --no-cache-dir --upgrade setuptools pip \
+ && python3.9 -m pip install --no-cache-dir \
       kubernetes \
       ansible==2.9.* \
       "molecule<3.3.0" \
```

I think this needs to be:

```suggestion
      ansible-core \
      molecule \
```